docs/tutorials/installiter8controller.md

```diff
@@ -1,22 +1,22 @@
 === "Helm"
     === "namespace scoped"
         ```shell
-        helm install --repo https://iter8-tools.github.io/iter8 iter8 traffic
+        helm install --repo https://iter8-tools.github.io/iter8 iter8 controller
         ```
 
     === "cluster scoped"
         ```shell
-        helm install --repo https://iter8-tools.github.io/iter8 iter8 traffic \
+        helm install --repo https://iter8-tools.github.io/iter8 iter8 controller \
         --set clusterScoped=true
         ```
     
 === "Kustomize"
     === "namespace scoped"
         ```shell
-        kubectl apply -k 'https://github.com/iter8-tools/iter8.git/kustomize/iter8/namespaceScoped?ref=v0.15.3'
+        kubectl apply -k 'https://github.com/iter8-tools/iter8.git/kustomize/controller/namespaceScoped?ref=v0.15.3'
```

Comment:
reference should change to v0.16.x for some x. Here, below and in deleteiter8controller.md